communities/asu-phoenix/community-impact-proposal.md

```diff
@@ -0,0 +1,123 @@
+        - Mobile Devs
+        - Video Game Developer Club
+        - Software Developer Association 
+        - Women in Computer Science 
+        - Technology + Innovation Lab 
+        - Sun Devil Robotics
+        - Intern Supply 
+        - PWN Devils 
+        - CodePath
+        - Desert Hacks
+        - Video Game Development 
+    - Digital Culture Organizations 
+        - Digital Culture Underground 
+        - [Other](https://artsmediaengineering.asu.edu/resources/students/clubs-organizations) 
+    - Engineering Organizations 
+        - Fulton Ambassadors 
+        - Fulton Student Council 
+        - Polytechnic Campus Orgs
+        - [Fulton Student Orgs](https://fso.engineering.asu.edu/student-org-directory/)
+
+- Phoenix Area, many partnerships + meetups 
+    - Galvanize, great co-working/event space
+    - #YesPHX Movement (Tech Startup Movement)
+    - Meetups 
+        - Code For America
+        - VR/AR Hangouts 
+        - IoT Meetups 
+    - Hackerrnest socials 
+    - Local Companies 
+        - GoDaddy 
+        - Intel 
+        - Amazon
+        - American Express 
+
+- **Culture**: There is a new excitement in the air, the tech scene is growing fast in the Phoenix Area and more people are getting incolced some way or the other even if not directly CS. 
+
+
+## Your Proposal
+
+Create a unifying way to showcase the existing organizations and tech influences to a community not only of veterns but the freshmen especially. This would involve hosting an event right after coming back from school. Currently there is a passport day that is successful to showcase ALL different clubs. There could possibly be a specific Tech centered one that really allows all those interested in Tech to see all the different opportunities of involvement they themselves can have in the ASU/Phoenix community. 
+
+
+- **Timing**: 
+
+| Week |Title|Description| 
+|:----:|:-------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------:|
+|   3  | Research Past Events/Choose Event Type | Understand which freshmen/beginner friendly events work best. Large Scale Hackathon? University Wide Tech Day? 
+|   4-5  |  Research/Recruitment of current  Freshmen/Sophomores, Faculty, Organizations      | We currently have joined forces with one hackathon team, start having people more involved |
+|   6  | Create a Timeline of event planning process | Have a structured plan and task list in order to assess what should be done over summer to have a successful back to school event|
+- **Community demographics**: On the school front, planning on reaching out to more organizations within the school and faculty members that are heavily involved. On the company front, reach out to past companies that were involved and be more involved with hackerrnest socials to discover new potentially interested community partners. 
```

Comment:
One thing that could be interesting here is exploring who your new freshmen are. Is there an online space where incoming freshmen gather before joining the school, like a Class of 2018 Fb page or something?